src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -49,20 +50,63 @@ ScatterUpdate::ScatterUpdate(const std::shared_ptr<ov::Node>& op, const GraphCon
           dataSize(0lu), indicesSize(0lu), axisSize(0lu),
           dataPrec(ov::element::undefined),
           indicesPrec(ov::element::undefined),
-          axisPrec(ov::element::undefined) {
+          axisPrec(ov::element::undefined),
+          ovOp(op) {
     std::string errorMessage;
     if (isSupportedOperation(op, errorMessage)) {
         errorPrefix = std::string(op->get_type_name()) + " node with name '" + getName() + "'";
     } else {
         OPENVINO_THROW_NOT_IMPLEMENTED(errorMessage);
     }
 
-    const auto node = ov::as_type_ptr<const ov::op::v12::ScatterElementsUpdate>(op);
-    if (node) {
-        reduction_type = node->get_reduction();
-        use_init_val = node->get_use_init_val();
-    } else {
-        reduction_type = ScatterUpdate::Reduction::NONE;
+    reduction_type = ScatterUpdate::Reduction::NONE;
+    if (const auto node_element = ov::as_type_ptr<const ov::op::v12::ScatterElementsUpdate>(op)) {
+        using OpReduction = ov::op::v12::ScatterElementsUpdate::Reduction;
```

Comment:
No need to change this case, due to `reduction_type` is already from `ScatterElementsUpdate`

Reply:
`reduction_type` was changed to use internal enum since ops might have different reductions (for example, `ScatterElementsUpdate` contains `mean`, `ScatterNDUpdate` contains `sub`), so switch would be required